src/NavItem/index.jsx

```diff
@@ -24,7 +24,7 @@ const NavItem = ( {
     forceHover,
     href,
     iconType,
-    isCurrentPage,
+    isCurrentCategory,
```

Comment:
Changing this prop name is a breaking change. So we need to keep both props for now and add a deprecation warning that is displayed when `isCurrentPage` is used (`console.warn`).

I'd also say that `isCurrentCategory` suffers from the same problem as `isCurrentPage`: it's too specific.

I think `isCurrent` might be the best name for this prop.

Reply:
@conor-cafferkey-sociomantic  done.